pom.xml

```diff
@@ -1252,7 +1252,7 @@
             <dependency>
                 <groupId>mysql</groupId>
                 <artifactId>mysql-connector-java</artifactId>
-                <version>5.1.48</version>
+                <version>8.0.22</version>
```

Comment:
This is likely to cause issues with other modules using MySQL.
If tests pass then it's fine otherwise I'd suggest to follow the approach in https://github.com/trinodb/trino/pull/7201/commits/ec64d121c8933fa9da366bd9f10f4240778d1a66 by keeping other modules at the existing version and doing those as a follow-up/separate task.

Reply:
I can try keeping `raptor-legacy` with `5.1.48`

Reply:
I agree, let's pin the old version in `raptor-legacy` only.
this should be commented like `<!-- TODO (link) update MySQL driver to version used everywhere else -->`

This should make complete the first commit. @guyco33 can you please do the change and extract to a separate PR, so that we can merge this?

Reply:
@findepi 
https://github.com/trinodb/trino/pull/8300